mnist/README.md

```diff
@@ -6,165 +6,96 @@ This example guides you through the process of taking an example model, modifyin
 
 Before we get started there a few requirements.
 
-### Kubernetes Cluster Environment
+## 1.Kubernetes Cluster Environment
 
-Your cluster must:
+For supported versions, please check Kubeflow document: https://www.kubeflow.org/docs/started/getting-started/
 
-- Be at least version 1.9
-- Have access to an S3-compatible object store ([Amazon S3](https://aws.amazon.com/s3/), [Google Storage](https://cloud.google.com/storage/docs/interoperability), [Minio](https://www.minio.io/kubernetes.html))
-- Contain 3 nodes of at least 8 cores and 16 GB of RAM.
-
-If using GKE, the following will provision a cluster with the required features:
-
-```
-export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
-gcloud alpha container clusters create ${USER} --enable-kubernetes-alpha --machine-type=n1-standard-8 --num-nodes=3 --disk-size=200 --zone=us-west1-a --cluster-version=1.9.3-gke.0 --image-type=UBUNTU
-```
-
-If using Azure, the following will provision a cluster with the required features, [using the az cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest):
```

Comment:
This section is compact enough that I feel there's some justification to keep it.  Even though it repeats in essence some of the info in the official linked guide, it's more to the point for the purposes of the example, and it also includes Azure instructions, which are currently missing from the official getting started guide.

Reply:
Hi, as @jlewi suggested, put things into right place. I'd like to put the azure instruction in kubeflow getting started document.

Reply:
+1